multisig/proposals/arbitrumone/vip-001/index.ts

```diff
@@ -0,0 +1,220 @@
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { arbitrumone } = NETWORK_ADDRESSES;
+
+const CHAINLINK_BTC_FEED = "0xd0C7101eACbB49F3deCcCc166d238410D6D46d57";
```

Comment:
Let's use the BTC/USD price feed from Chainlink, instead of the WBTC/USD price feed.

```suggestion
const CHAINLINK_BTC_FEED = "0x6ce185860a4963106506C203335A2910413708e9";
```